pkg/lclient/containers.go

```diff
@@ -33,10 +48,9 @@ func (dc *Client) GetContainerList() ([]types.Container, error) {
 // containerConfig - configurations for the container itself (image name, command, ports, etc) (if needed)
 // hostConfig - configurations related to the host (volume mounts, exposed ports, etc) (if needed)
 // networkingConfig - endpoints to expose (if needed)
-// containerName - name to give to the container
 // Returns an error if the container couldn't be started.
-func (dc *Client) StartContainer(containerConfig *container.Config, hostConfig *container.HostConfig, networkingConfig *network.NetworkingConfig, containerName string) error {
-	resp, err := dc.Client.ContainerCreate(dc.Context, containerConfig, hostConfig, networkingConfig, containerName)
+func (dc *Client) StartContainer(containerConfig *container.Config, hostConfig *container.HostConfig, networkingConfig *network.NetworkingConfig) error {
+	resp, err := dc.Client.ContainerCreate(dc.Context, containerConfig, hostConfig, networkingConfig, "")
```

Comment:
any reason why Container Name is empty string? 🤔

Reply:
Good question. As we use labels to identify and retrieve containers for odo-docker, we don't care too much about the container name. By setting it to be a blank string, it allows Docker to assign a randomly generated name to it instead and avoid any potential conflicts

Reply:
yeah i thought so too, that is what you were going for hmm